app/page.tsx

```diff
@@ -239,7 +239,7 @@ export default function Home() {
           <Grid item xs={12} md={6}>
             <Grid
               container
-              sx={{ paddingBottom: "70px", overflow: "visable" }}
+              sx={{ paddingBottom: "50px", overflow: "visable" }}
```

Comment:
`overflow: "visable"` this is not going to work due to spelling mistake :P

Reply:
fixing!